ansible_base/lib/abstract_models/common.py

```diff
@@ -37,13 +37,10 @@ def get_url_for_object(obj, request=None):
 
 
 class CommonModel(models.Model):
-    # These are fields that should be reversed lookup as related fields.
-    # For example, an environment has related organizations so environment might specify reverse_foreign_key_fields = ['organizations']
-    # This would end up with a view like environment/1/organizations
-    reverse_foreign_key_fields = []
-
     # Any field marked as encrypted will automatically be stored in an encrypted fashion
     encrypted_fields = []
+    # Any field set in here will not be used in the views
+    ignore_relations = []
```

Comment:
This conflicts with the solution to the same problem

https://github.com/ansible/django-ansible-base/blob/devel/test_app/models.py#L42

```python
    teams_with_no_view = models.ManyToManyField(Team, related_name='related_fields_test_model_teams_with_no_view')
    teams_with_no_view.related_view = None
```

So that test_app model should be updated, and maybe it wasn't being tested correctly to begin with.

Reply:
And down below, just in this file, delete:

```python
        for field in self._meta.many_to_many:
            if getattr(field, 'related_view', False) is None:
                # Give fields a chance to opt out of showing up here by forcing related_view to None
                continue
```

I think your solution will work for both ForeignKey and ManyToMany, which the prior solution did not, so :+1: on that solution, just delete the old approach.